src/main/webapp/resources/js/components/samples/SampleDetailViewer.tsx

```diff
@@ -114,32 +189,31 @@ function DisplaySampleDetails({ sampleId, projectId, removeSample, children }: D
  * React component to provide redux store to sampledetailviewer
  * @param sampleId - identifier for a sample
  * @param projectId - identifier for a project
- * @param removeSample - function to remove the sample from the cart.
+ * @param displayActions - Whether to display add to/remove from cart buttons. Displayed by default
  * @param children
  * @returns {JSX.Element}
  * @constructor
  */
 
 export interface SampleDetailViewerProps {
-    sampleId: number;
```

Comment:
If you merge in dev, you can do something like this, I like it because it is super clear what you are doing:
```typescript
export interface SampleDetailViewerProps {
  sampleId: Pick<Sample, "id">;
  projectId: Pick<Project, "id">;
  displayActions?: boolean;
  children: React.ReactNode;
}
```

Reply:
Updated in e7628892ff95d6c2b3905eb2892f5cffdc192f56